Classes/ViewHelpers/Outlet/FormViewHelper.php

```diff
@@ -0,0 +1,65 @@
+<?php
+namespace FluidTYPO3\Flux\ViewHelpers\Outlet;
+
+    /*
+     * This file is part of the FluidTYPO3/Flux project under GPLv2 or later.
+     *
+     * For the full copyright and license information, please read the
+     * LICENSE.md file that was distributed with this source code.
+     */
+use FluidTYPO3\Flux\Provider\AbstractProvider;
+
+/**
+ */
+class FormViewHelper extends \TYPO3\CMS\Fluid\ViewHelpers\FormViewHelper
+{
+
+    /**
+     * @var AbstractProvider
+     */
+    protected $provider;
+
+    /**
+     * @var array
+     */
+    protected $record;
+
+    /**
+     * Initialize arguments.
+     *
+     * @return void
+     */
+    public function initialize()
+    {
+        $this->provider = $this->viewHelperVariableContainer->get(self::class, 'provider');
+        $this->record = $this->viewHelperVariableContainer->get(self::class, 'record');
+
+        if (!$this->hasArgument('extensionName')) {
+            $this->arguments['extensionName'] = $this->viewHelperVariableContainer->get(self::class, 'extensionName');
+        }
+
+        if (!$this->hasArgument('pluginName')) {
```

Comment:
That's a good question - I think that depends on a lot of other variables, and I even think it's different on LTS vs. master since `render()` method arguments are all but deprecated.